api/src/main/java/org/motechproject/nms/api/web/OpsController.java

```diff
@@ -80,4 +92,59 @@ public void startUpkeep() {
         LOGGER.info("/upkeep");
         eventRelay.sendEventMessage(new MotechEvent(KilkariConstants.SUBSCRIPTION_UPKEEP_SUBJECT));
     }
+
+    @RequestMapping(value = "/addFlw",
+            method = RequestMethod.POST,
+            headers = { "Content-type=application/json" })
+    @ResponseStatus(HttpStatus.OK)
+    @Transactional
+    public void createUpdateFlw(@RequestBody AddFlwRequest addFlwRequest) {
+        // TODO: obscure phone number?
```

Comment:
Yes we should obscure the phone number, but it doesn't look like it is being logged so I think this comment can be removed.